app/src/main/res/layout/fragment_edit_categories.xml

```diff
@@ -39,20 +39,11 @@
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintTop_toTopOf="@id/my_sayings_title" />
 
-    <View
-        android:id="@+id/top_view"
-        android:layout_width="0dp"
-        android:layout_height="2dp"
-        android:background="@color/colorPrimary"
-        app:layout_constraintEnd_toEndOf="@id/edit_categories_view_pager"
-        app:layout_constraintStart_toStartOf="@id/edit_categories_view_pager"
-        app:layout_constraintTop_toTopOf="@id/edit_categories_view_pager" />
-
     <androidx.viewpager2.widget.ViewPager2
         android:id="@+id/edit_categories_view_pager"
         android:layout_width="0dp"
         android:layout_height="0dp"
-        android:layout_marginTop="@dimen/edit_category_pager_margin_top"
+        android:layout_marginTop="12dp"
```

Comment:
I generally don't like using dimens, but since the app is already using them, we should probably just change this dimen instead of hardcoding it, for consistency